### PR TITLE
Chore: Alter /develop index page and refine text

### PR DESCRIPTION
**Pull request by NoSQLKnowHow**
Refining the text of the new /develop index page, adding links, and more context to this critical page.

### CODE REVIEW

**Review by eaddingtonwhite — DISMISSED** [review]

File: docs/develop/index.md [hunk 1]
```diff
@@ -10,49 +11,67 @@ import { SdkExampleTabs } from "@site/src/components/SdkExampleTabs";
...
 ## Client configuration objects
 
-You will also need to create a `Configuration` object to configure your Momento Cache client. `Configuration` objects control things such as timeouts, retries, logging, middleware, and more.
+`Configuration` objects contain the settings necessary for how a Momento client should connect to Momento services. The `Configuration` object controls settings such as timeouts, retries, logging, middleware, and more.
+
+Each SDK contains pre-built `Configuration` objects to help get you up and running as quickly as possible. We did the hard work of tuning for various deployment environments so you can focus on the things unique to your business. (We even have a blog series about it! [Shockingly simple: Cache clients that do the hard work for you](https://www.gomomento.com/blog/shockingly-simple-cache-clients-that-do-the-hard-work-for-you))
+
+Pre-built configurations come with a `latest()` version, which will always give you the latest recommended configuration for a given environment. For example,
+
+`configuration: Configurations.InRegion.Default.latest(),`
```
Comment (cprice404) [inline docs/develop/index.md]: If you use the SdkExampleTabs widget for this then we will have the syntax available in the other languages besides just JS, but, up to you

**Review by cprice404 — COMMENTED** [review]
one nit about using the react widget for the code samples but up to you

File: docs/develop/index.md (hunk 1 above)
Comment (NoSQLKnowHow) [reply docs/develop/index.md]: Do we have to add something to the SDK repos to show this line?

Comment (cprice404) [reply docs/develop/index.md]: I have the v1s already:

https://github.com/momentohq/client-sdk-javascript/blob/main/examples/nodejs/doc-examples-js-apis.ts#L45-L55

but yeah you would need to add one for `latest`.

i'm writing docs for this code injection stuff now.

Comment (NoSQLKnowHow) [reply docs/develop/index.md]: ok, this is fixed.

**Review by cprice404 — APPROVED** [review]

**Pull request merged**